pyop2/base.py

```diff
@@ -998,6 +1065,12 @@ def __str__(self):
     def __repr__(self):
         return "MixedSet(%r)" % (self._sets,)
 
+    def __eq__(self, other):
+        try:
+            return self._sets == other._sets
+        except AttributeError:
```

Comment:
I think it is better be cautious with exception handling since one might catch exceptions coming from unexpected places. For example, if `self` or `other` have a `_sets` property whose implementation is bugged, or calls some bugged code, e.g. typo in the attribute name of a completely unrelated object, then that exception will be caught here, causing bugs that are very confusing to debug.